pom.xml

```diff
@@ -232,7 +232,7 @@
         <version.org.opensearch.latest>2.4.1</version.org.opensearch.latest>
 
         <version.com.google.code.gson>2.9.1</version.com.google.code.gson>
-        <version.software.amazon.awssdk>2.18.1</version.software.amazon.awssdk>
+        <version.software.amazon.awssdk>2.19.8</version.software.amazon.awssdk>
```

Comment:
@marko-bekhta this is not just a build/test dependency, it's exposed in published artifacts.

Can you please create a separate issue on JIRA and update the issue key in the commit that performs this update?
Also please mention that new issue in this PR title/description :)

Reply:
sir yes sir 😄

Reply:
Thank you sir :bow: